packages/banner/package.json

```diff
@@ -22,6 +22,8 @@
     "@hig/icons": "^1.0.0",
```

Comment:
You should also update `icons` to the latest version and remove `themes` as I think that is an older version of `theme-context`, also remove `classnames`.